### PR TITLE
Docs/Cloud: Add "Asia Pacific, Singapore" to regions

### PR DESCRIPTION
**Pull request by joselcvarela**
## Description
Within Docs, add "Asia Pacific, Singapore" to the Regions options list on Cloud section.

### This is how it looks like:
<blockquote>
Add Asia Pacific, Singapore to regions
</blockquote>

## Type of Change

- [x] Docs

## Requirements Checklist

N/A

If adding a new feature:

N/A

### CODE REVIEW

File: docs/cloud/glossary.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ via the schema endpoints. Enterprise clients: the Directus Team will work with y
 
 The location you choose to host your Project can have measurable impact. To optimize your Project and meet any local
 data compliance laws such as GDPR, Standard and Enterprise, you can choose the region your Project is hosted in. There
-are 19 Datacenter Regions available to Enterprise Projects and two available to Standard Projects.
+are 19 Datacenter Regions available to Enterprise Projects and three available to Standard Projects: United States,
+East; Europe, Frankfurt; Asia Pacific, Singapore.
 
 ![Datacenter Regions](https://cdn.directus.io/docs/v9/cloud/glossary/glossary-20220322A/datacenter-region-20220331A.webp)
```
Comment (joselcvarela) [inline docs/cloud/glossary.md:71]: Can you take a new screenshot @erondpowell ?
I figure I don't have the same screen resolution to take a new screenshot.

Comment (erondpowell) [reply docs/cloud/glossary.md]: With this table, I think that image I had is no longer necessary so I deleted it.

**Review by erondpowell — APPROVED** [review]

**Comment by erondpowell** [comment]
@rijkvanzanten btw are we cool to merge these docs despite the linter test and the Netlify stuff going haywire lately?

File: docs/cloud/glossary.md (hunk 1 above)
Comment (erondpowell) [reply docs/cloud/glossary.md]: thanks for this dude!

**Comment by rijkvanzanten** [comment]
Yeah that's ok 👍🏻

**Pull request merged**